backends/p4test/CMakeLists.txt

```diff
@@ -50,11 +50,31 @@ set (P4TEST_SUITES
   "${P4C_SOURCE_DIR}/testdata/p4_16_samples/*.p4"
   "${P4C_SOURCE_DIR}/testdata/p4_16_errors/*.p4"
   )
+# Builds a list of tests for which P4Info generation is not supported. It makes
+# sense not to fail the test just because P4Info generation fails. It probably
+# would not make sense to run all the tests twice (one with and one without
+# P4Runtime).
+set (P4RUNTIME_EXCLUDE
+  # Two counter instances provided for psa_direct_counter, illegal according to
+  # PSA spec and fails in P4Info generation
+  testdata/p4_16_samples/psa-counter5.p4
+  # Need to convert PSA_MeterColor_t to a serializable enum
+  testdata/p4_16_samples/psa-meter1.p4
+  # Compiler Bug: Type_Name is not a canonical type, use getTypeType()?
+  testdata/p4_16_samples/pvs-nested-struct.p4
```

Comment:
Can you please file an issue for these showing how to reproduce them?

Reply:
yes will do. I was going to try and get a backtrace, but if you can take a look that would be great